R/metadata.R

```diff
@@ -0,0 +1,46 @@
+do_plotting_metadata <- function(country) {
+  metadata <- naomi::get_plotting_metadata(country)
```

Comment:
Lines 3-7 can be replaced with (I believe):

```
lapply(split(metadata, metadata$data_type), build_data_type_metadata)
```

and a similar pattern will probably help simplify the functions below

Reply:
Thank you! I wanted something like that, so much neater.